index.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module 'rlp' {
+    import { Buffer } from 'safe-buffer'
+
+    type EncodeScalarInput = Buffer|string|number
+    type EncodeInput = EncodeScalarInput|Array<EncodeScalarInput>
+
+    export function encode(input: EncodeInput): Buffer
+    export function decode(input: EncodeInput): Array<Buffer>
```

Comment:
`decode` seems to have an optional parameter for `stream`, which means it returns a "streaming output", e.g. a the decoded data and the remainder buffer.

I'm fine not exposing it as that complicates the definition and not sure if anybody uses it.